internal/identity/oidc/gitlab/gitlab.go

```diff
@@ -8,12 +9,20 @@ import (
 	"net/url"
 
 	oidc "github.com/coreos/go-oidc"
+	"golang.org/x/oauth2"
+
 	"github.com/pomerium/pomerium/internal/httputil"
+	"github.com/pomerium/pomerium/internal/identity"
+	pom_oidc "github.com/pomerium/pomerium/internal/identity/oidc"
 	"github.com/pomerium/pomerium/internal/sessions"
 	"github.com/pomerium/pomerium/internal/version"
-	"golang.org/x/oauth2"
 )
 
+var _ identity.Authenticator = &GitLabProvider{}
+
+// Name identifies the GitLab identity provider
+const Name = "gitlab"
```

Comment:
@desimone Good job! What do you think about adding `Name` into the `consts` declared below [here](https://github.com/pomerium/pomerium/pull/560/files#r398806555), to help improve readability. What do you think?

Reply:
Good idea.